### ISSUE TITLE
spec error in config

### ISSUE THREAD

**Issue by tommy-mor** [ykskb/phrag#8]
here are the errors, they are fairly large:

https://gist.github.com/tommy-mor/3f6ef52a98dd742b2fe9cee33fb7b98f

the exception is from doing ``(phrag.core/schema (phrag.context/options->config {:db {:connection db}}))``

using postgres. gonna peek at this, but would like some help. library seems dope, hope to use it :)

**Comment by tommy-mor** [comment]
first gist is the config, this one is the full stacktrace and spec error: 
https://gist.github.com/tommy-mor/f3575055dce4f469db6a867e036aeaa8

**Comment by ykskb** [comment]
Hello @tommy-mor ,

Yea spec error from Lacinia gets huge and it's hard to tell exactly where it's failing.

Would it be possible for you to share DB schema somehow? It might be data type mapping Phrag is missing.

**Comment by tommy-mor** [comment]
yes, I have just emailed it to you at the address on your personal site

**Comment by ykskb** [comment]
Received and thanks! I'll have a look when I can. I hope to look into it soon but it might take some time as I'm currently occupied with some stuff.

**Comment by tommy-mor** [comment]
No worries. It's not blocking anything,  but it would be very nice
